File: core/src/main/java/com/arij/core/controllers/SimpleRestController.java
package com.arij.core.controllers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class SimpleRestController {

    @GetMapping("/hello")
    public String index() {
        return "Greetings from Spring Boot!";
    }

}

File: core/src/main/java/com/arij/core/controllers/SimpleRestController.kt
package com.arij.core.controllers

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestParam
import org.springframework.web.bind.annotation.RestController

@RestController
class SimpleRestController {

    @GetMapping("/hello")
    fun hello(@RequestParam(value="name", defaultValue = "World") name: String): String {
        return "Hello, $name!"
    }

}
